### PR TITLE
Update bundle_id_ID.properties

### PR DESCRIPTION
**Pull request by Kanaede**
Small update [ID]

### CODE REVIEW

File: core/assets/bundles/bundle_id_ID.properties [hunk 1]
```diff
@@ -146,21 +146,21 @@ mod.multiplayer.compatible = [gray]Bisa Digunakan dalam Multiplayer
...
-mod.incompatiblemod = [red]Incompatible
-mod.blacklisted = [red]Unsupported
+mod.incompatiblegame = [red]Game kadaluarsa
+mod.incompatiblemod = [red]Tidak kompatibel
+mod.blacklisted = [red]Tidak didukung
 mod.unmetdependencies = [red]Unmet Dependencies
 mod.erroredcontent = [scarlet]Konten Mengalami Kesalahan
 mod.circulardependencies = [red]Circular Dependencies
-mod.incompletedependencies = [red]Incomplete Dependencies
+mod.incompletedependencies = [red]Ketergantungan tidak lengkap
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_id_ID.properties:155]: dependen is more accurate; ketergantungan refers to individual's mental state

File: core/assets/bundles/bundle_id_ID.properties [hunk 2]
```diff
@@ -54,25 +54,25 @@ mods.github.open-release = Laman Rilis
...
 schematic.copy.import = Impor dari papan klip
 schematic.shareworkshop = Bagikan di Workshop
-schematic.flip = [accent][[{0}][]/[accent][[{1}][]: Balik Bagan
-schematic.saved = Bagan telah disimpan.
-schematic.delete.confirm = Bagan ini akan benar-benar dihapus.
-schematic.edit = Edit Schematic
+schematic.flip = [accent][[{0}][]/[accent][[{1}][]: Balik Skema
+schematic.saved = Skema telah disimpan.
+schematic.delete.confirm = Skema ini akan benar-benar dihapus.
+schematic.edit = Ubah Skema
```
Comment (DaGamerFiles) [inline core/assets/bundles/bundle_id_ID.properties:73]: Sunting is specific; Ubah is too broad

**Review by DaGamerFiles — COMMENTED** [review]
not bad so far, although string amount is not the same as main bundle and some that are changed need revision, just let me know if you want to continue this and I'll help you or go with mine which have been updated but have not been uploaded

File: core/assets/bundles/bundle_id_ID.properties (hunk 2 above)
Comment (Kanaede) [reply core/assets/bundles/bundle_id_ID.properties]: Fixed

File: core/assets/bundles/bundle_id_ID.properties (hunk 1 above)
Comment (Kanaede) [reply core/assets/bundles/bundle_id_ID.properties]: Fix, I also changed the one below it

File: .vscode/settings.json
Comment (DaGamerFiles) [inline .vscode/settings.json:1]: ayo keep this one to yourself

File: fastlane/metadata/steam/indonesian/empy
Comment (DaGamerFiles) [inline fastlane/metadata/steam/indonesian/empy:1]: rename empy to `short-description.txt`, I will take care of the rest here soon

**Review by DaGamerFiles — CHANGES_REQUESTED** [review]
PR creates merge conflict, need fix

File: .vscode/settings.json
Comment (Kanaede) [reply .vscode/settings.json:1]: fixed

File: fastlane/metadata/steam/indonesian/empy
Comment (Kanaede) [reply fastlane/metadata/steam/indonesian/empy:1]: okay

**Comment by DaGamerFiles** [comment]
PR will be moved to a new one, OP have asked to retire from translating.

**Pull request closed without merging**